log-parser/cpp/CMakeLists.txt

```
@@ -29,7 +29,13 @@ endif()
 set(CMAKE_CXX_STANDARD 17)
 
 # Process Abseil's CMake build system
-add_subdirectory(abseil-cpp)
+include(FetchContent)
+FetchContent_Declare(
+	abseil-cpp
+	GIT_REPOSITORY https://github.com/abseil/abseil-cpp.git
+	GIT_TAG 78be63686ba732b25052be15f8d6dee891c05749 # version 20230125.0
```

Nit. I like to just have the tag after `#`.
```suggestion
	GIT_TAG 78be63686ba732b25052be15f8d6dee891c05749 # 20230125.0
```